### PR TITLE
Added sound setting per chat room

### PR DESCRIPTION
**Pull request by poradz**
image
image

### CODE REVIEW

**Comment by mariuszkrzaczkowski** [comment]
## Migration Recommended

It appears you are still using the CoPilot OAuth App - we recommend you [migrate to the GitHub App](https://github.com/apps/blackduck-copilot/installations/new/permissions?suggested_target_id=10201412&repository_ids[]=28060246)

**Comment by mariuszkrzaczkowski** [comment]
## Black Duck Security Report

Merging #11156 into developer will not change security risk.

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/YetiForceCompany/YetiForceCRM/pull-requests/11156)

File: modules/Chat/actions/ChatAjax.php [hunk 1]
```diff
@@ -54,13 +54,15 @@ public function getChatConfig(App\Request $request)
...
-			'refreshTimeGlobal' => App\Config::module('Chat', 'REFRESH_TIME_GLOBAL'),
-			'showNumberOfNewMessages' => App\Config::module('Chat', 'SHOW_NUMBER_OF_NEW_MESSAGES')
+			'config' => [
+				'isChatAllowed' => \App\User::getCurrentUserRealId() === \App\User::getCurrentUserId(),
+				'isDefaultSoundNotification' => \App\Config::module('Chat', 'DEFAULT_SOUND_NOTIFICATION'),
+				'refreshMessageTime' => App\Config::module('Chat', 'REFRESH_MESSAGE_TIME'),
+				'refreshRoomTime' => App\Config::module('Chat', 'REFRESH_ROOM_TIME'),
+				'maxLengthMessage' => App\Config::module('Chat', 'MAX_LENGTH_MESSAGE'),
+				'refreshTimeGlobal' => App\Config::module('Chat', 'REFRESH_TIME_GLOBAL'),
+				'showNumberOfNewMessages' => App\Config::module('Chat', 'SHOW_NUMBER_OF_NEW_MESSAGES')
```
Comment (rskrzypczak) [inline modules/Chat/actions/ChatAjax.php]: ?

File: app/Chat.php [hunk 2]
```diff
@@ -316,20 +316,24 @@ public static function getRoomsByUser(?int $userId = null)
...
 	{
 		$numberOfNewMessages = 0;
 		$roomInfo = static::getRoomsByUser();
+		$roomList = [];
 		foreach (['crm', 'group', 'global'] as $roomType) {
-			foreach ($roomInfo[$roomType] as $item) {
-				if (isset($item['cnt_new_message'])) {
-					$numberOfNewMessages += $item['cnt_new_message'];
+			foreach ($roomInfo[$roomType] as $room) {
+				if (isset($room['cnt_new_message'])) {
```
Comment (rskrzypczak) [inline app/Chat.php]: ?

**Pull request merged**